content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -1,68 +1,102 @@
 # Sharing Experiments
 
-Use `dvc exp push` and `dvc exp pull` to share experiments. This works like
-[sharing regular project versions][sharing-data], but here DVC takes care of
-synchronizing to/from both Git and [DVC remotes][remote storage] as needed:
+To share experiments, you can:
```

Comment:
`To share something you can ... view ...` - it took me a while to parse and see if there is something missing :)